File: fixtures/simple-iface/tests/bindings/test_simple_iface.swift
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

import uniffi_simple_iface

let obj = makeObject(inner: 9000)
assert(obj.getInner() == 9000)

File: fixtures/simple-iface/tests/bindings/test_simple_iface.py
# This Source Code Form is subject to the terms of the Mozilla Public
# License, v. 2.0. If a copy of the MPL was not distributed with this
# file, You can obtain one at http://mozilla.org/MPL/2.0/.

from uniffi_simple_iface import *

obj = make_object(9000)
assert obj.get_inner() == 9000
obj.some_method()
